data/json/recipes/other/tool.json

```diff
@@ -374,6 +374,18 @@
     "autolearn": true,
     "components": [ [ [ "colander_steel", 1 ], [ "wire_mesh", 1 ], [ "splatter_guard", 1 ] ] ]
   },
+  {
+    "type": "recipe",
+    "result": "sponge",
```

Comment:
I have some doubt that you can make a sponge by putting a rag, some plastic and dry grass into a cooking pot.
If you want to add a recipe for a cleaning tool you should probably find one that is actually craftable and add it to the game with it's recipe.